packages/webvh/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@credo-ts/webvh",
+  "version": "0.5.16",
+  "description": "Credo WebVH DID method implementation",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "files": [
+    "build"
+  ],
+  "publishConfig": {
+    "main": "build/index",
+    "types": "build/index",
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "pnpm run clean && pnpm run compile",
+    "clean": "rimraf -rf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "pnpm run build",
+    "test": "jest",
+    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand --config jest.config.js"
+  },
+  "dependencies": {
+    "@credo-ts/anoncreds": "workspace:*",
+    "@credo-ts/askar": "workspace:*",
+    "@credo-ts/core": "workspace:*",
+    "@credo-ts/node": "workspace:*",
```

Comment:
This dependency might imply issues when running on React Native. Are you sure you need it?

Reply:
I am using `new AskarWallet(` which requires `new agentDependencies.FileSystem()` as a parameter which is the only time this is being used. I think I'm either not supposed to use AskarWallet as I am or there is a way to get around this?

Reply:
What about using this.agentContext.wallet instead?